tests/fixtures.py

```diff
@@ -69,6 +69,16 @@ class OptionalIfNotFormTest(forms.Form):
     content = forms.TextAreaField(__("Content"))
 
 
+class RequiredIfFormTest(forms.Form):
+    blurb = forms.TextAreaField(__("Blurb"), validators=[forms.validators.RequiredIf('content')])
```

Comment:
In general, `RequiredIf` should reference earlier fields as that is how the user will see them.